Source/MipsExecutor.cpp

```diff
@@ -364,7 +364,7 @@ void CMipsExecutor::PartitionFunction(uint32 functionAddress)
 	{
 		uint32 currentPoint = -1;
 		for(PartitionPointSet::const_iterator pointIterator(partitionPoints.begin());
-			pointIterator != partitionPoints.end(); pointIterator++)
+			pointIterator != partitionPoints.end(); ++pointIterator)

```

Comment:
Humm, dunno why I said that, but never mind, you can't use a range for loop here.